governance/polls/Add GUNIV3DAIUSDCFT with 0.01% Fee Tier as a new Vault Type - November 29, 2021.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Add GUNIV3DAIUSDCFT with 0.01% Fee Tier as a new Vault Type - November 29, 2021
```

Comment:
PE have confirmed the ilk will be called GUNIV3DAIUSDC2-A :)

Reply:
Would probably phrase this as "Add GUNIV3DAIUSDC2-A as a new Vault Type"

Reply:
Make sure to update the main poll title too!

Reply:
Awesome. Thanks for getting in touch with PE! Fixed!